src/components/DonateButton.jsx

```diff
@@ -1082,23 +1082,38 @@ export default class Root extends React.PureComponent {
 
   afterSuccessfulDonate() {
     const { donateToDefaultDac } = this.state;
+    const { customThanksMessage } = this.props.model;
+
+    const el = document.createElement('div');
+    el.innerHTML = customThanksMessage;
 
     if (!this.props.currentUser || this.props.currentUser.name) {
       // known user
       if (donateToDefaultDac) {
-        React.swal({
-          title: 'Thank you!',
-          text: 'Would you like to support Giveth as well?',
-          icon: 'success',
-          buttons: ['No Thanks', 'Support Giveth'],
-        }).then(result => {
-          if (result) {
-            this.defaultDacDonateButton.current.openDialog();
-          }
-        });
+        if (typeof customThanksMessage !== 'undefined') {
```

Comment:
`customThanksMessage` is shown only if we have `donateToDefaultDac`

Reply:
donateToDefualtDac means that user has no profile. If he has no profile, we suggest him to fill his profile. For user without profile who donate to custom message campaign we have the following options.
1. show custom message with more info e.g. Common Stack
2. suggest him to fill his profile (as my commit)
3. show more info e.g Common Stack offer and suggest him to fill his profile

It seems to me  (2) is the best option. (3) suggesting two things is complicated UX. (1) is low priority comparing to (2). It is my opinion, what do you think?

Reply:
- if know user{
-       if custom thanks
-          show custom thanks
-       elsif Donate to Default
-          suggest donate to default 
-        else 
-           show simple thanks
- else //anon user
-     suggest fill profile

Reply:
Yes it's the correct scenario

Reply:
@aminlatifi  please check out  my new commit.